templates/trusted-node/prover-config.json

```diff
@@ -143,7 +143,7 @@
      * aggregatorClientHost is the hostname that this prover will
      * attempt to use while connecting to the aggregator
      */ -}}
-    "aggregatorClientHost": "zkevm-node-aggregator{{.deployment_suffix}}",
```

Comment:
I think we still want to support the `zkevm-node` so we might need to change the aggregator client host value based on `sequencer_type`.

You can use something like:

```go
{{if eq .sequencer_type "zkevm-node"}}
"aggregatorClientHost": "zkevm-node-aggregator{{.deployment_suffix}}",
{{else if eq .sequencer_type "erigon"}}
"aggregatorClientHost": "cdk-node{{.deployment_suffix}}",
{{end}}
```

Reply:
Good one, I wonder for how long are we going to support zkevm node, at least in CDK we're not supporting it anymore. But easy to add this.